blockchain/treasury_test.go

```diff
@@ -1844,6 +1844,262 @@ func TestTSpendExpendituresPolicy(t *testing.T) {
 	g.AcceptTipBlock()
 }
 
+// TestExpendituresReorg tests that the correct treasury balance is tracked
+// when reorgs remove tspends/tadds from the main chain. Test plan:
+//
+// - Approve and mine TSpend and TAdd
+// - Reorg to chain without TSpend/Tadd
+// - Mine until TSpend/TAdd would have been mature
+// - Extend TSpend/TAdd chain and reorg back to it
+func TestExpendituresReorg(t *testing.T) {
+	// Use a set of test chain parameters which allow for quicker vote
+	// activation as compared to various existing network params.
+	params := quickVoteActivationParams()
+
+	// Clone the parameters so they can be mutated, find the correct deployment
+	// for the fix sequence locks agenda, and, finally, ensure it is always
+	// available to vote by removing the time constraints to prevent test
+	// failures when the real expiration time passes.
+	const tVoteID = chaincfg.VoteIDTreasury
+	params = cloneParams(params)
+	tVersion, deployment, err := findDeployment(params, tVoteID)
+	if err != nil {
+		t.Fatal(err)
+	}
+	removeDeploymentTimeConstraints(deployment)
+
+	// Dave off tvi and mul.
```

Comment:
Save?

Reply:
I thought it was a joke from Marco :P

Fixed all instances.